js/main.js

```diff
@@ -1,3 +1,6 @@
 import {createArrayData} from './data.js';
+import './popup.js';
```

Comment:
По возможности лучше не делать модулей с сайд эффектами или чем-то что там делается само без вызова. А напрямую экспортировать их модуля набор функций с помощью которых можно будет им управлять, например так
```suggestion
import {createPopup} from './popup.js';
```

Иначе если нам понадобится функционал этого модуля в другом месте, то мы его туда тоже импортируем и что, логика повторится? или не повторится? не должно быть ничего неявного, все в функции, кроме определения глобальных переменных

Reply:
Есть исключения, когда подключаешь что-то вроде стилей или подобного, где неважно количество, место, порядок, но в любом случае явные вызов и исользование любого из модулей всегда будет предпочтительнее простого импорта